src/widgets/courses-school/ui/courses.scss

```diff
@@ -58,6 +58,7 @@
           img {
```

Comment:
Please use .module.scss and

Reply:
https://t.me/c/2142251410/1519

Reply:
I've just added one line to existed scss file. Or you desided to move whole project on modules and I should change every style file that I'll touch to them?